GrandCentralBoard/Widgets/Harvest/HarvestWidget.swift

```diff
@@ -12,15 +12,21 @@ import GCBCore
 final class HarvestWidget: WidgetControlling {
 
     private let widgetView = HarvestWidgetView.fromNib()
+    private let numberOfDays: Int
 
     let sources: [UpdatingSource]
 
     var view: UIView {
         return widgetView
     }
 
-    init(sources: [UpdatingSource]) {
+    init(sources: [UpdatingSource], numberOfDays: Int) {
         self.sources = sources
+        self.numberOfDays = numberOfDays
+
+        let emptyCircleChartModel = CircleChartViewModel(startAngle: 0, items: [CircleChartItem(color: UIColor.gcb_blackColor(), ratio: 1.0)])
+        widgetView.configureWithViewModel(HarvestWidgetViewModel(lastDayChartModel: emptyCircleChartModel,
```

Comment:
💡 Minor thing (feel free to ignore), but this might be more verbose if written this way:
```swift
let emptyWidgetViewModel = HarvestWidgetViewModel(lastDayChartModel: emptyCircleChartModel,
                                                  lastNDaysChartModel: emptyCircleChartModel,
                                                  numberOfLastDays: numberOfDays)
widgetView.configureWithViewModel(emptyWidgetViewModel)
```
Nesting is often bad for readability.